README.md

```diff
@@ -58,6 +58,98 @@ running Go 1.9 or higher.
 
 - [Getting Started with Azure Blob Service in Go](https://github.com/Azure-Samples/storage-blob-go-getting-started)
 
+# Generate code
```

Comment:
One suggestion that @amarzavery had, that I really liked would be to put instructions like this in a `documentation` folder. It stays discoverable, is available offline, and doesn't pollute the front-page README that folks hit, and gets versioned with the code base. Some stuff still belongs in this project's wiki, but generation instructions that are specific to tooling at a moment in time feel natural for source control. :)

Reply:
Fixed